roles/ngi_reports/tasks/dependencies.yml

```diff
@@ -30,10 +30,13 @@
   template: src="texlive.profile.j2" dest="{{ ngi_pipeline_conf }}/texlive.profile" mode=0664
   tags: ngi_reports 
 
+- shell: "find {{ texLive_dest }}/src/install* -maxdepth 0 -type d"
```

Comment:
Note that a problem with all these solutions is that if you're running this in the future, and somehow suddenly `http://mirror.ctan.org/systems/texlive/tlnet/{{ texLive_file }}` is bundling a new version, then you'll end up with several folders that match your expression due to the wildcard. 

If it is good enough to catch the latest created folder under the `src/` subdir, then perhaps you could do something like `shell: ls -1 -td {{ texLive_dest }}/src/install* | head -1`